tests/vcs/git/GitCommitBearTest.py

```diff
@@ -288,6 +343,57 @@ def test_body_checks(self):
                              body_regex=r'TICKER\s*CLOSE\s+[1-9][0-9]*'), [])
         self.assert_no_msgs()
 
+        # Testing invalid emails based on format only
+        self.git_commit('Shortlog\n\n'
+                        'invalid#!format!*!@email.com\n'
+                        'another.*invalid@format.co\n'
+                        'Fix 2017')
+        message = 'Body contains these invalid emails:\n'
+        message += 'invalid#!format!*!@email.com\n'
+        message += 'another.*invalid@format.co'
+
+        self.assertEqual(self.run_uut(), [message])
+
+        # Return Object for successful mock
+        class ReturnObject:
+            ERROR_CODES = {}
+
+
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/vcs/git/GitCommitBearTest.py
+++ b/tests/vcs/git/GitCommitBearTest.py
@@ -357,7 +357,6 @@
         # Return Object for successful mock
         class ReturnObject:
             ERROR_CODES = {}
-
 
         # Test successful DNS lookup
         with unittest.mock.patch("bears.vcs.git.GitCommitBear.is_email", return_value=ReturnObject()):
```